instagram/db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
Metrics/LineLength: Line is too long. [103/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)